### ISSUE TITLE
App Review - Log in issues

### ISSUE THREAD

**Issue by gesabo** [pinterest/ios-pdk#141]
Is anyone else having problems getting through app review as far as logging in?  I updated my app but did nothing to the log-in code and they're telling me they can't log in, yet I log in fine on iPhone 6s, iPad Air, and the simulator? I can't figure out the disconnect.  All of my P List settings are correct as far as I can tell?

**Comment by levi** [comment]
@gesabo thanks for bringing this up. This is the first time I have heard of this. Are you able to successfully login on your local build? Also, is the account you're providing the App Store reviewers the same one you're using to test locally?

**Comment by gesabo** [comment]
Thanks @levi for the quick reply...every so often, not every time, when I tapped my log in button, i wasn't directed to the Pinterest URL to authenticate, no error triggered however, so I'm not sure why.   I was able to get through review by (1) removing private permissions I was requesting and (2) I was using about a 6 month old version of the SDK, whether this had anything to do with it or not who knows, app review is such a dice roll, maybe I just got a different reviewer.  In any event, closing this issue since I was approved.
